internal/cli/atlas/security/ldap/save.go

```diff
@@ -64,6 +67,7 @@ func (opts *SaveOpts) newLDAPConfiguration() *atlas.LDAPConfiguration {
 			AuthorizationEnabled:  opts.authorizationEnabled,
 			Hostname:              opts.hostname,
 			Port:                  opts.port,
+			UserToDNMapping:       []*atlas.UserToDNMapping{{Match: opts.mappingMatch, LDAPQuery: opts.mappingLdapQuery, Substitution: opts.mappingSubstitution}},
```

Comment:
`UserToDNMapping` is an array, you can only define one value here, do you think other users would agree with this?

This may also be the problem with our test you are now passing an empty element in the array when no flag is set so adding to the array may need to be conditioned

Also since is an array I wonder if it may then make sense to let users pass multiple values

Reply:
@gssbzn that's a good point. Based on [the docs|https://docs.mongodb.com/manual/reference/configuration-options/#mongodb-setting-security.ldap.userToDNMapping], having multiple values seems like a reasonably well supported scenario. I've never used multiple values. I do think other users may need them. 

I think it would make sense to move this out of the "atlas security ldap save" command and into a separate new "atlas security ldap addMapping" command (or just "add" to match the "atlas security ldap delete" command? Or even better, replace "delete" with "deleteMapping" that takes an argument of the match string of the entire entry to delete).

I can't work on that in the near term though. I think supporting one mapping is better than none.